schedule.py

```diff
@@ -0,0 +1,54 @@
+from typing import Iterable
+from flask_restful import abort
+
+from scheduledao import ScheduleDAO
+from event import Event
+
+class Schedule:
+    dao = ScheduleDAO()
+
+    def __init__(self, owner: str, events: Iterable[Event]=[]):
+        self.owner = owner
+        self.events = set(events)
+
+    @classmethod
+    def fromdict(cls, sched_dict):
+        owner = sched_dict['owner']
+        events = [Event.get(e) for e in sched_dict['events']]
+        return cls(owner, events)
+
+    def add_event(self, event):
+        self.events.add(event)
+
+    def remove_event(self, event_name):
+        self.events = {e for e in self.events if e.name != event_name}
+
+    def serialize(self):
```

Comment:
not needed

Reply:
Refer to my comment under `Event.serialize()`